### PR TITLE
[BLOG] The Polars vs pandas difference nobody is talking about

### PR DESCRIPTION
**Pull request by MarcoGorelli**
## Text styling

- [ ] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [ ] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [ ] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [ ] Blog post featured image is in PNG or JPEG format, **not** SVG.
- [ ] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/dataframe-group-by.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+Python lambda function.
+
+On the other hand, if you innovate on the API side, you can enable new possibilities. There's a reason
+that
+
+> I came for the speed, but stayed for the syntax
+
+is a common refrain among Polars users.
+
+There may be a more general lesson here: if you have the courage to do things differently, you may be rewarded.
```
Comment (gabalafou) [inline apps/labs/posts/dataframe-group-by.md:164]: This line doesn't really connect for me with anything else in the blog post, seems to come out of nowhere.

File: apps/labs/posts/dataframe-group-by.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+If you'd like to learn about how to use Polars effectively, or how to solve problems in your organisation
+using Polars, Quansight is here to help - you can get in touch [here](https://quansight.com/about-us/#bookacallform).
```
Comment (gabalafou) [inline apps/labs/posts/dataframe-group-by.md:167]: I wonder if there's some way we could or should set this off from the rest of the blog post, something like the following (not saying we should do it exactly like this):

And now, a small message from our fellow coworkers at Quansight Consulting... If you'd like to learn about how to use Polars effectively, or how to solve problems in your organisation
using Polars, please [get in touch](https://quansight.com/about-us/#bookacallform).

**Comment by gabalafou** [comment]
Note: I changed the base branch of this PR to reflect the new blog publishing process. Once this is approved, it will be merged directly into the main branch and deployed live to the Web.

**Review by pavithraes — APPROVED** [review]
Thanks, Marco. The post reads good to me, I think the views-sales example works. :)

**Pull request merged**